src/screens/home/TokenImage.tsx

```diff
@@ -277,3 +279,24 @@ export const getIconSource = (
       return undefined
   }
 }
+
+export const getTokenByChainId = (
```

Comment:
`getTokenSymbolByChainId` will be more appropriate name for this function as that's what it returns.

Reply:
I agree, done.